### DISCUSSION TITLE
Try websockets

### DISCUSSION THREAD

**Discussion by mbsteixeira** [uNetworking/uWebSockets.js#224] [Q&A] [answered]
I have the C/C+ uWebsockets running fine.
When a try the uWebsockets.js,  there is a line in host.cpp  with the statement   uWS::Loop::defaultLoop().
Well,  in the source Loop.h  there is no method called defaultLoop.
Where can I find it?

**Comment by ghost** [comment]
So you don't want to run it inside node.js?

**Comment by mbsteixeira** [comment]
Yes, I'd like run out of node.js. Is it possible?

**Comment by ghost** [comment, marked as answer]
Yes you can use uWS.js as a stand alone thing but you would have to do a lot of work yourself. Host.cpp is really only a proof of concept

**Comment by mbsteixeira** [comment]
Thanks!
